src/Service/EloquaApiClient.php

```diff
@@ -270,7 +274,7 @@ public function getBaseUrl() {
   private function doBaseUrlRequest() {
     // TODO Ideally merge all the Guzzle requests into one generic method.
     // Guzzle Client.
-    $guzzleClient = new GuzzleClient([
+    $guzzleClient = $this->httpClientFactory->fromOptions([
```

Comment:
surely this shouldnt be `$guzzleClient` as per the line 46 where its `protected $httpClientFactory;`